static/js/zamboni/buttons.js

```diff
@@ -179,16 +178,28 @@ var installButton = function() {
         };
     });
 
+    var showDownloadAnyway = function($button) {
+        var $installShell = $button.parents('.install-shell');
+        var $downloadAnyway = $installShell.find('.download-anyway');
+        if ($downloadAnyway.length) {
+            var href = $button.filter(':visible').attr('href');
+            var pos = href.lastIndexOf('/');
+            href = escape_(href.slice(0, pos) + '/type:attachment/' + href.slice(pos + 1));
```

Comment:
I really don't have a better alternative but this essentially transforms `/firefox/downloads/latest/privacy-badger17/platform:2/addon-506646-latest.xpi` into `/firefox/downloads/latest/privacy-badger17/platform:2/type:attachment/addon-506646-latest.xpi` right?